src/interfaces/records-write.ts

```diff
@@ -708,6 +714,13 @@ export class RecordsWrite implements MessageInterface<RecordsWriteMessage> {
       entryId   : await RecordsWrite.getEntryId(this.author, this.message.descriptor)
     };
 
+    // we don't want the tags properties to occupy the descriptor index namespace
```

Comment:
Had to read multiple times to understand the meaning of "occupy the descriptor index namespace", I think.

Consider:
```suggestion
    // in order to avoid name clashes with system-level index keys
```